MiguelAngelVidalLiborio/Dia9-02Septiembre/vet/src/main/java/com/veterinary/exception/ModelNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package com.veterinary.exception;
+
+public class ModelNotFoundException extends RuntimeException {
```

Comment:
It's not so common to extend from RuntimeExc... but I think this is a very good example... if there is no DB, how to recover from that kind of err.... good!

PS. Maybe when app is loading context, you should validate that DB already exists... only a comment, not a requirement for this excercise